pixeltable/catalog/catalog.py

```diff
@@ -1529,6 +1536,67 @@ def _load_tbl(self, tbl_id: UUID) -> Optional[Table]:
         self._tbls[tbl_id] = view
         return view
 
+    def _load_tbl_with_specified_version(self, tbl_id: UUID, effective_version: int) -> Optional[Table]:
```

Comment:
Chatty name. load_tbl_at_version?

Also: the parameter should be 'version', we only use effective_version if it's Optional[int].